media-api/app/controllers/MediaApi.scala

```diff
@@ -226,6 +226,21 @@ class MediaApi(
     }
   }
 
+  def softDeleteImage(id: String) = auth.async { request =>
+    implicit val r = request
+
+    elasticSearch.getImageById(id) map {
+      case Some(_) =>
+        val updateMessage = UpdateMessage(subject = SoftDeleteImage, id = Some(id),
+          softDeletedMetadata = Some(SoftDeletedMetadata(
+            DateTime.now(DateTimeZone.UTC),
+            request.user.accessor.identity)))
+        messageSender.publish(updateMessage)
```

Comment:
no need for val as case class name describes what it is, named params on the SoftDeletedMetadata args and changed line breaks & indentation to make things clearer  
```suggestion
        messageSender.publish(
          UpdateMessage(
            subject = SoftDeleteImage, 
            id = Some(id),
            softDeletedMetadata = Some(SoftDeletedMetadata(
              deleteTime = DateTime.now(DateTimeZone.UTC),
              deletedBy = request.user.accessor.identity
            )
          )
        )
```

Reply:
@twrichards will update and squash the PR now